HISTORY.rst

```diff
@@ -31,6 +31,7 @@ v0.5.0 (TBD)
 * In ``w3microdata`` ``_extract_properties``, ``_extract_property_refs``, 
   ``_extract_property``, ``_extract_property_value`` and ``_extract_item``
   now need ``items_seen`` and ``url`` to be passed as arguments.
+* Add the option in ``extruct.extract`` to return HTML node with the result of metadata extraction.
```

Comment:
Can you split this line so it is no longer than 79 columns? :)

Reply:
Agree, I'll fix both.
By the way concerning these 79 columns, do we follow this rule in the whole project? I noticed that there are also some other lines in both code and text files that are longer than 79 columns. I know that these 79 columns are specified by PEP8 standard but my personal opinion about that is that 120 characters is more optimal and 79 characters are too less for current screen resolutions.
In addition, I can see in PEP8 that the rules are even more restricted and "for flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.".
What do you guys think? What rules do we accept?

Reply:
@jakubwasikowski we're not strict about it, but I think a convention which we follow is to use 79 columns - both for code and docstrings, unless there is a reason not to do so. 

Regarding "reason": introducing a local variable to make an expression fit to a limit is fine; splitting a string constant with a long URL just to fit it to 79 columns is not a good idea.

It may also depend on a project: it is fine to use 120 columns limit if you notice that the project doesn't respect 79 columns limit often; if code is mostly fit into 79 columns, it is better to go with that limit, so that the code is consistent - unless there is a strong reason to diverge.